src/Reviews/components/ReviewListItem.jsx

```diff
@@ -4,22 +4,36 @@ import ax from 'axios';
 class ReviewListItem extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      helpfulness: this.props.review.helpfulness,
+      updatedHelpfulness: false,
+      reported: false,
+    };
     this.helpfulReview = this.helpfulReview.bind(this);
+    this.reportReview = this.reportReview.bind(this);
   }
 
   helpfulReview(reviewID) {
+    if (this.state.updatedHelpfulness) { return; }
     ax.put(`/api/reviews/${reviewID}/helpful`)
       .then((res) => { console.log('put helpful'); })
```

Comment:
We can probably leave this console log out, but that can also be done on next pass, since we're still in dev.